src/ziggurat/messaging/producer.clj

```diff
@@ -103,9 +103,20 @@
   "Get exponential backoff enabled for specific channel from config."
   (-> (ziggurat-config) :stream-router topic-entity :channels channel :retry :exponential-backoff-enabled))
 
-(defn- get-exponential-backoff [retry-count message-retry-count queue-timeout-ms]
+(defn- get-exponential-backoff [topic-entity channel message-payload]
```

Comment:
rename this function, it does not express intent. you can call it something like `backoff-exponent`.

Reply:
this is updated